src/main/java/dk/cngroup/lentils/repository/HintTakenRepository.java

```diff
@@ -22,6 +22,16 @@
             nativeQuery = true)
     List<HintTaken> findAllByTeamAndCypher(Long teamId, Long cypherId);
 
+    @Query(value = "SELECT ht FROM HintTaken ht " +
+            "LEFT JOIN Hint h ON ht.hint.hintId = h.hintId " +
+            "WHERE h.cypher.cypherId=?1 ORDER BY h.value ASC")
+    List<HintTaken> findAllByCypher(Long cypherId);
+
+    @Query(value = "SELECT ht FROM HintTaken ht " +
+            "LEFT JOIN Hint h ON ht.hint.hintId = h.hintId " +
+            "WHERE ht.team.teamId=?1 ORDER BY h.value ASC")
+    List<HintTaken> findAllByTeam(Long teamId);
```

Comment:
Je tady to @Query nutné? U metody pod touto to není a vypadá to že to funguje :D

Reply:
pravda, diky ;)